qa/tasks/mgr/dashboard/test_health.py

```diff
@@ -4,19 +4,51 @@
 from .helper import DashboardTestCase
 
 
-class DashboardTest(DashboardTestCase):
+class HealthTest(DashboardTestCase):
     CEPHFS = True
 
-    def test_health(self):
-        data = self._get("/api/dashboard/health")
+    def test_minimal_health(self):
+        data = self._get("/api/health/minimal")
+        self.assertStatus(200)
+
+        self.assertNotIn('audit_log', data)
+        self.assertIn('client_perf', data)
+        self.assertSortedEqual(
+            data['client_perf'].keys(),
+            ['read_bytes_sec', 'read_op_per_sec',
+             'recovering_bytes_per_sec', 'write_bytes_sec',
+             'write_op_per_sec']
+        )
+        self.assertNotIn('clog', data)
+        self.assertIn('df', data)
+        self.assertEqual(data['df'].keys(), ['stats'])
+        self.assertSortedEqual(
+            data['df']['stats'].keys(),
+            ['total_avail_bytes', 'total_bytes', 'total_objects',
+             'total_used_bytes']
+        )
+        self.assertIn('fs_map', data)
+        self.assertSortedEqual(
+            data['fs_map'].keys(), ['filesystems', 'standbys'])
+        for fs in data['fs_map']['filesystems']:
+            self.assertEqual(fs.keys(), ['mdsmap'])
```

Comment:
Well, that's not the same. `self.assertIn('mdsmap', ['mdsmap', 'foo'])` is also `True`.

Reply:
Well, that's true, but since we are testing against a list that only contains one item ``assertIn()`` will fit our needs.

``self.assertEqual(fs.keys(), ['mdsmap'])`` will only succeed if ``keys()`` returns a list with ONE item, so ``assertIn`` can be used here, otherwise the original test itself is incorrect.

Reply:
I think we specifically want to test for equality here; the minimal health data should not have any other keys in a 'filesystems' object aside from 'mdsmap'